Stylesheet/css/typography.css

```diff
@@ -0,0 +1,35 @@
+@import url('https://fonts.googleapis.com/css2?family=Bree+Serif&family=Rubik&display=swap');
+/* font-family: 'Bree Serif', serif;
+font-family: cursive;
+font-family: 'Rubik', sans-serif; */
+
+h1 {
+    font-family: var(--st-header-text);
+    font-weight: 700;
+    font-size: 2.5rem;
+}
+
+h2 {
+    font-family: var(--st-subheader-text);
+    font-weight: 600;
+    font-size: 2rem;
+}
+
+h3 {
+    font-family: var(--st-subheader-text);
+    font-weight: 500;
+    font-size: 1.5rem;
+}
+
```

Comment:
Generally `p` tag and `h4` tag have same font size. Only differ in font weight. you are giving same font weight to both of them

Reply:
Okay changing it